specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/examples/VpnConnectionUpdate.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
I don't see this example being referenced anywhere.

Reply:
Removing